modules/modules.nix

```diff
@@ -25,6 +25,13 @@ let
     (loadModule ./config/i18n.nix { condition = hostPlatform.isLinux; })
     (loadModule ./files.nix { })
     (loadModule ./home-environment.nix { })
+    (loadModule ./i18n/input-method/default.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./i18n/input-method/fcitx.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./i18n/input-method/fcitx5.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./i18n/input-method/hime.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./i18n/input-method/nabi.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./i18n/input-method/uim.nix { condition = hostPlatform.isLinux; })
+    (loadModule ./i18n/input-method/kime.nix { condition = hostPlatform.isLinux; })
```

Comment:
You can use `imports = [ ..... ]` inside `i18n/input-method/default.nix`.

Reply:
I am not sure how i should do that here, could you please give me an example?

Reply:
In `i18/input-method/default.nix`:

``` nix
{ config, pkgs, ... }:

{
  imports = [
    ./fcitx.nix
    ./fcitx5.nix
    ./hime.nix
    ./nabi.nix
    ./uim.nix
    ./kime.nix
  ];

  options.i18n.inputMethod = ...;
  
  config = ...;
}
```